grails-app/views/project/index.gsp

```diff
@@ -70,7 +70,22 @@
                         <a href="${createLink(controller: 'transcribe', action: 'index', id: projectInstance.id)}" class="btn btn-primary btn-lg" role="button">Get Started <span class="glyphicon glyphicon-arrow-right"></span></a>
                         <g:if test="${projectInstance.tutorialLinks}">
                             <a href="${(projectInstance.tutorialLinks ? '#tutorial' : createLink(controller: 'tutorials', action: 'index'))}" class="btn btn-lg btn-hollow ${oldClass} tutorial">View tutorial</a>
-                            <div id="tutorialContent" class="hidden">${raw(projectInstance.tutorialLinks)}</div>
+                            <div id="tutorialContent" class="hidden">
+                                <h4>Expedition Tutorial Information</h4>
```

Comment:
Likely just my unfamiliarity with DigiVol but I found the distinction between tutorial links and tutorial list unclear.  It would also be nice to display a message under the headings for the 0 tutorial case e.g. https://digivol-test.ala.org.au/project/index/6306